docs/user-guide/7-Lucene-search.rst

```diff
@@ -293,6 +293,23 @@ The above query returns the favorite objects of specific user.
 
     Only system administrator with "manage_users" capabilities can search for other users favorites.
 
+Comment author field (\ ``comment_author:``\ )
+------------------------------------------------------------
+
+Typing the field ``comment_author:`` you can search for objects commented by selected user.
+
+.. code-block::
+
+   comment_author:<user login>
+
+The above query returns the objects commented by user <user login>.
+
+.. warning::
+
+    Remember that you can only search using logins for users registered in mwdb.
+    Otherwise you receive error massage: ``No such user: <login>``.
```

Comment:
Isn't it a bit obvious? :thinking:  Maybe something like:

```
Comment authors are kept only for existing users, so you can't search for comments from deleted accounts.
```

The second warning sounds OK :+1: